peer/pendingheap/config.go

```diff
@@ -48,7 +48,6 @@ type Configuration struct {
 //        http:
 //          url: https://host:port/rpc
 //          fewest-pending-requests:
-//            capacity: 25
```

Comment:
Did you mean to delete this line? The configuration field is still there.

Reply:
Yup, that configuration line would likely confuse more people than it would help. see https://github.com/yarpc/yarpc-go/commit/daf6346cb861e212e13cdf9ce6b3002541a36448